tidb-cloud/_index.md

```diff
@@ -10,151 +10,139 @@ summary: TiDB Cloud is a fully-managed Database-as-a-Service (DBaaS) that brings
 
 </LearningPath>
 
 <LearningPath label="Maintain" icon="cloud7">
 
-[Create a Cluster](https://docs.pingcap.com/tidbcloud/create-tidb-cluster)
+[Create a Cluster](https://docs.tidb.io/tidbcloud/create-tidb-cluster-serverless/)
 
-[Connect to a Cluster](https://docs.pingcap.com/tidbcloud/connect-to-tidb-cluster)
+[Connect to a Cluster](https://docs.tidb.io/tidbcloud/connect-to-tidb-cluster-serverless/)
 
-[Use an HTAP Cluster](https://docs.pingcap.com/tidbcloud/tiflash-overview)
+[Use an HTAP Cluster](https://docs.tidb.io/tidbcloud/tiflash-overview)
 
-[Back Up and Restore Data](https://docs.pingcap.com/tidbcloud/backup-and-restore)
+[Back Up and Restore Data](https://docs.tidb.io/tidbcloud/backup-and-restore-serverless/)
 
-[Scale a Cluster](https://docs.pingcap.com/tidbcloud/scale-tidb-cluster)
+[Use API (Beta)](https://docs.tidb.io/tidbcloud/api-overview)
 
-[Pause or Resume a TiDB Cluster](https://docs.pingcap.com/tidbcloud/pause-or-resume-tidb-cluster)
-
-[Stream Data](http://docs.pingcap.com/tidbcloud/changefeed-overview)
-
-[Use API (Beta)](https://docs.pingcap.com/tidbcloud/api-overview)
-
-[Use TiDB Cloud CLI](https://docs.pingcap.com/tidbcloud/get-started-with-cli)
+[Use TiDB Cloud CLI](https://docs.tidb.io/tidbcloud/get-started-with-cli)
 
 </LearningPath>
 
 <LearningPath label="Migrate" icon="cloud3">
 
-[Import Sample Data](https://docs.pingcap.com/tidbcloud/import-sample-data)
-
-[From MySQL](https://docs.pingcap.com/tidbcloud/migrate-data-into-tidb)
-
-[From Amazon Aurora MySQL](https://docs.pingcap.com/tidbcloud/migrate-from-aurora-bulk-import)
-
-[From Amazon S3 or GCS](https://docs.pingcap.com/tidbcloud/migrate-from-amazon-s3-or-gcs)
+[From Amazon RDS for Oracle](https://docs.tidb.io/tidbcloud/migrate-from-oracle-using-aws-dms/)
 
-[From TiDB Self-Managed](https://docs.pingcap.com/tidbcloud/migrate-from-op-tidb)
+[Import Sample Data](https://docs.tidb.io/tidbcloud/import-sample-data-serverless/)
 
-[From CSV Files](https://docs.pingcap.com/tidbcloud/import-csv-files)
+[Import CSV Files](https://docs.tidb.io/tidbcloud/import-csv-files-serverless/)
 
-[From Apache Parquet Files](https://docs.pingcap.com/tidbcloud/import-csv-files)
+[Import Parquet Files](https://docs.tidb.io/tidbcloud/import-parquet-files-serverless/)
 
-[With MySQL CLI](https://docs.pingcap.com/tidbcloud/import-with-mysql-cli)
+[With MySQL CLI](https://docs.tidb.io/tidbcloud/import-with-mysql-cli-serverless/)
 
 </LearningPath>
 
 <LearningPath label="Monitor" icon="cloud6">
 
-[Status and Metrics](https://docs.pingcap.com/tidbcloud/monitor-tidb-cluster)
+[Status and Metrics](https://docs.tidb.io/tidbcloud/monitor-tidb-cluster)
 
-[Built-in Monitoring](https://docs.pingcap.com/tidbcloud/built-in-monitoring)
+[Built-in Monitoring](https://docs.tidb.io/tidbcloud/built-in-monitoring)
 
-[Built-in Alerting](https://docs.pingcap.com/tidbcloud/monitor-built-in-alerting)
+[Built-in Alerting](https://docs.tidb.io/tidbcloud/monitor-built-in-alerting)
 
```

Comment:
```suggestion
```

Reply:
In the https://docs.tidb.io/tidbcloud/monitor-built-in-alerting/ doc, there is a note: 
![image](https://github.com/user-attachments/assets/40b67479-a095-4d58-9a8b-f940e85a9cd1)

Reply:
Yes. This file is removed in https://github.com/pingcap/docs/pull/20594. I'll commit the suggestion.